### ISSUE TITLE
Ouput disapperes when it is 3 or 24!!!

### ISSUE THREAD

**Issue by samir-aldroubi** [cms-dev/cms#783]
Hi,
I was adding a problem to my local cms but it couldn't be evaluated. I checked the test cases and then noticed that 2 test cases output files were empty(The output of theses test cases was 3 and 24). I tried adding multiple test cases with (3,24) as an output in multiple problems and for multiple input data and the same problem occurred again(output file is empty).

Can you please check that out?

Thanks in advance.

**Comment by lerks** [comment]
Could you clarify? Are you saying that you were adding a new testcase to a task through the AdminWebServer and that the reference output file for that testcase has, as content, "3" or "24" then the file that was actually stored by CMS was empty? I just tried this and on my machine it works correctly.

**Comment by samir-aldroubi** [comment]
Yes, that's precisely what happened. It is really weird!

**Comment by lerks** [comment]
Since I am unable to reproduce it I don't really know how to debug this. Is it still happening with the latest master branch? If so could you provide all relevant logs? Maybe a screen recording to show the precise steps you've taken? A (partial) dump of your database to make sure your state isn't corrupted. Anything would help...

**Comment by newey** [comment]
@samir-aldroubi I've encountered the same issue. This occurred when I manually deleted parts of the file cache to save space. Though since CMS stores files by their hash, it was unable to locate the files on disk, this caused them to be loaded as empty files. My hacky fix was to append spaces to the end of the files that were problematic, this changes their hash and causes them to be stored again. Though the issue can be properly fixed by clearing your file caches.
